galaxy_ng/app/api/ui/serializers/collection.py

```diff
@@ -65,7 +65,7 @@ def get_signatures(self, obj):
         data = []
         for signature in obj.signatures.all():
             sig = {}
-            sig["signature"] = bytes(signature.data).decode("utf-8")
+            sig["signature"] = signature.data
```

Comment:
@mdellweg why did I have to make this change? Looking at the data coming from console dot, the signatures still show up as bytes, but when i sign a collection locally it shows up as a string.

Reply:
Figured it out: https://github.com/pulp/pulp_ansible/pull/845

Reply:
We save the signature as ascii armored text. It was a pain to handle it as bytes. Also import-export couldn't cope with it at all.